**Input**

In [1]:
input

<bound method Kernel.raw_input of <ipykernel.ipkernel.IPythonKernel object at 0x00000184FF37D160>>

- function
- bound method

In [2]:
import os
os.getcwd

<function nt.getcwd()>

In [4]:
input()

 10


'10'

In [6]:
input('Enter your Name: ')

Enter your Name:  Sunil


'Sunil'

In [7]:
input

<bound method Kernel.raw_input of <ipykernel.ipkernel.IPythonKernel object at 0x00000184FF37D160>>

In [8]:
input('What is your city: ')


What is your city:  Bnagalore


'Bnagalore'

In [13]:
input('Enter Value :    ')


Enter Value :     sdkjefhqkjhfiqu4oi3451-05 odpomeqemw9i230123


'sdkjefhqkjhfiqu4oi3451-05 odpomeqemw9i230123'

In [1]:
input()
input()
input()

 hello
 how are you
 good


'good'

In [2]:
input('Enter the name : ')
input('Enter the age : ')
input('Enter the city : ')

Enter the name :  Python
Enter the age :  20
Enter the city :  Bangalore


'Bangalore'

In [3]:
Name=input('Enter the name : ')
Age=input('Enter the age : ')
City=input('Enter the city : ')
print(Name)
print(Age)
print(City)

Enter the name :  Python
Enter the age :  30
Enter the city :  Bangalore


Python
30
Bangalore


In [4]:
n1=input('Enter the number1 : ')
n2=input('Enter the number2 : ')
n1+n2

Enter the number1 :  20
Enter the number2 :  30


'2030'

In [6]:
n1=int('20')
n2=int('30')
n1+n2

n1=int(input('Enter the number1 : '))
n2=int(input('Enter the number2 : '))
n1+n2


50

In [7]:
n1=int(input('Enter the N1 : '))
n2=int(input('Enter the N2 : '))
n1+n2

Enter the N1 :  100
Enter the N2 :  100.5


ValueError: invalid literal for int() with base 10: '100.5'

In [12]:
n1=int(input('Enter the N1 : '))
n2=float(input('Enter the N2 : '))
round(n1+n2,0)

Enter the N1 :  34
Enter the N2 :  67.90


102.0

## Eval

**To convert the int/float values as per resultes**

- Ex:
       - n1=int(input('Enter the N1 : ')) # 100
       - n2=float(input('Enter the N2 : ')) #100.50
       - n1+n2 # Ouput: 200.5

       - n1=int(input('Enter the N1 : ')) # 100
       - n2=float(input('Enter the N2 : ')) #100
       - n1+n2 # Ouput: 200.0  # here we need to expect value is 200 ,Hence we are using Keyword is **Eval**

In [15]:
n1=eval(input('Enter the N1 : '))
n2=eval(input('Enter the N2 : '))
n1+n2

Enter the N1 :  100
Enter the N2 :  10


110

In [1]:
n1=eval(input('Enter the N1 : '))
n2=eval(input('Enter the N2 : '))
n1+n2


Enter the N1 :  10
Enter the N2 :  10


20

In [29]:
#wsp ask the user enter 3 number
#find the avg

n1=eval(input('Enter the N1: '))
n2=eval(input('Enter the N2: '))
n3=eval(input('Enter the N3: '))
n4=(n1+n2+n3)/3
print(round(n4,4))

Enter the N1:  10.55
Enter the N2:  20.70
Enter the N3:  30.40


20.55


In [23]:
n1=eval(input('Enter the N1: '))
n2=eval(input('Enter the N2: '))
n3=eval(input('Enter the N3: '))
n4=(n1+n2+n3)/3
print(round(n4))


Enter the N1:  10.6
Enter the N2:  20.5
Enter the N3:  20.5


17


In [25]:
p=eval(input('Enter the Pi : '))
r=eval(input('Enter the Radius : '))
Area = p*r*r
print(Area)


Enter the Pi :  3.14
Enter the Radius :  20


1256.0


In [26]:

b=eval(input('Enter the breadth : '))
h=eval(input('Enter the height : '))
Area_of_rectangle=float(0.5)*b*h
Area_of_rectangle

Enter the breadth :  10
Enter the height :  10


50.0

In [27]:
b=eval(input('Enter the breadth : '))
l=eval(input('Enter the length : '))
rectangle=b*l
rectangle

Enter the breadth :  10
Enter the length :  20


200

In [38]:
b=eval(input('Enter the bill amount: '))
t=eval(input('Enter the tip amount: '))
Total_amount = b+t
print(Total_amount)


Enter the bill amount:  2000
Enter the tip amount:  220


2220


In [39]:
s=eval(input('Enter the salary amount: '))
t=eval(input('Enter the tax amount: '))
Total_tax = s/t*100
print(Total_tax)

Enter the salary amount:  20000
Enter the tax amount:  20


100000.0
